packages/kuma-gui/src/app/kuma/locales/en-us/index.yaml

```diff
@@ -81,6 +81,17 @@ common:
     none: '—'
     created: 'Created'
     modified: 'Modified'
+components:
+  status-badge:
+    tooltip:
+      not_available: |
+        The online or offline status of this entity is unknown
+  x-empty-state:
+    title: 'No data'
+    body: ''
+    action:
+      label: ''
+      href: ''
```

Comment:
These are specific to our application, not `x`. They could have gone in `kuma` or `service-mesh` but seeing as `kuma` already had a locales file I dumped them in there. I'm tempted to combine `kuma` and `service-mesh` at some point which would make one less module.